tangelo/linq/translator/translate_sympy.py

```diff
@@ -212,15 +212,19 @@ def translate_op_to_sympy(qubit_operator, n_qubits):
         sympy.core.add.Add: Summation of sympy.physics.quantum.TensorProduct
             objects.
     """
-    from sympy import Identity
-    from sympy.physics.paulialgebra import Pauli
+    from sympy import Matrix, I, zeros
     from sympy.physics.quantum import TensorProduct
 
     # Pauli string to sympy Pauli algebra objects.
-    map_to_paulis = {"I": Identity(1), "X": Pauli(1), "Y": Pauli(2), "Z": Pauli(3)}
+    map_to_paulis = {
```

Comment:
What were the consequences of this changes, out of curiosity ?

Reply:
Instead of getting 
```
sin(theta/2)**2*<11|*sigma2xsigma2xsigma1xsigma1*|11> + ...
```
for an expectation value, it is applying the operations (<11|11>=1, and so on, and we get an imaginary number as an expectation value).

Reply:
Great, thanks.